onnx/shape_inference/implementation.h

```diff
@@ -33,14 +33,14 @@ class SymbolTableImpl : public SymbolTable {
  public:
   SymbolTableImpl() : index_(0) {}
 
-  void addFromGraph(const GraphProto& g) {
+  void addFromGraph(const GraphProto& g) override {
     AddExistingSymbolicDims(g.input());
     AddExistingSymbolicDims(g.output());
     AddExistingSymbolicDims(g.value_info());
   }
   // Creates a new unique symbol with the given prefix and adds it to the SymbolTable
   // Returns the newly created symbol
-  std::string createNew(const std::string& symbol_prefix = "unk__") {
+  std::string createNew(const std::string& symbol_prefix = "unk__") override {
```

Comment:
Lint: default arguments on virtual or override methods are prohibited. Can be fixed separately

Reply:
Why separately? I think it will break code that calls this function without a parameter (which does exist). Can we move the default argument to the parent class?